Repository manquant dans la configuration Maven

Une erreur dans Maven empèche la compilation depuis le merge de #9.
```[ERROR] Failed to execute goal on project SimpleAuthJava:
 Could not resolve dependencies for project fr.esgi:SimpleAuthJava:jar:1.0.0-SNAPSHOT:
 The following artifacts could not be resolved:
 commons-logging:commons-logging:jar:99.0-does-not-exist, commons-logging:commons-logging-api:jar:99.0-does-not-exist, log4j:log4j:jar:99-empty:
 Failure to find commons-logging:commons-logging:jar:99.0-does-not-exist in https://repo.maven.apache.org/maven2 was cached in the local repository,
 resolution will not be reattempted until the update interval of central has elapsed or updates are forced -> [Help 1]

Caused by: org.eclipse.aether.transfer.ArtifactNotFoundException:
 Failure to find commons-logging:commons-logging:jar:99.0-does-not-exist in https://repo.maven.apache.org/maven2 was cached in the local repository,
 resolution will not be reattempted until the update interval of central has elapsed or updates are forced
```

Maven semble ne pas trouver les dépendances 99x.

--- a/pom.xml
+++ b/pom.xml
@@ -210,4 +210,18 @@
             <version>99-empty</version>
         </dependency>
     </dependencies>
+
+    <repositories>
+        <repository>
+            <id>jboss-3rd-party-releases</id>
+            <name>JBoss 3rd-party Releases</name>
+            <url>https://repository.jboss.org/nexus/content/repositories/thirdparty-releases/</url>
+            <layout>default</layout>
+        </repository>
+        <repository>
+            <id>version99-qos</id>
+            <name>Version99 QoS.ch</name>
+            <url>http://version99.qos.ch/</url>
+        </repository>
+    </repositories>
 </project>

